src/gnssro/gnssro_bufr2ioda.f90

```diff
@@ -381,7 +381,9 @@ program gnssro_bufr2ioda2
 
 call check( nf90_def_var(grpid_metadata, "ascending_flag",  NF90_INT, nlocs_dimid, varid_asce))
 call check( nf90_put_att(grpid_metadata, varid_asce, "longname", "the original occultation ascending/descending flag" ))
-call check( nf90_put_att(grpid_metadata, varid_asce, "valid_range", "0/descending or 1/ascending" ))
+call check( nf90_put_att(grpid_metadata, varid_asce, "valid_range", int((/ 0, 1 /))) )
+call check( nf90_put_att(grpid_metadata, varid_asce, "flag_values", int((/ 0, 1 /))) )
+call check( nf90_put_att(grpid_metadata, varid_asce, "flag_meanings", "descending ascending") )
```

Comment:
This is very nice!

Reply:
> src/ncar-bufr2nc-fortran/gnssro_mod.f90 is out-of-sync with gnssro_bufr2ioda.f90. A few more changes are needed to bring it up-to-date. Is there a need to maintain src/ncar-bufr2nc-fortran?

Thanks again, @jamiebresch. I just fixed the other warning for `qfro` there. I'll leave the question about the maintenance of `src/ncar-bufr2nc-fortran` for the others because I'm not the right person to say about it.